.env

```diff
@@ -1,4 +0,0 @@
-###> symfony/framework-bundle ###
```

Comment:
Why deleting this file ?

Reply:
Please ignore this file in the pull request, I just slipped up.